demo/views/demo-all.dust

```diff
@@ -46,14 +46,11 @@
   <input type="text" class="offleft">
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
```

Comment:
Using `<script src="/carbon-components.min.js"></script>` works for me. I didn't need to add `CarbonComponents.settings.disableAutoInit = false;` either.